src/components/Comments/CommentItem.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react';
+import { Comment } from '../../types/Comment';
+import { useAppDispatch } from '../../app/hooks';
+import {
+  deleteCommentById,
+  filterComment,
+} from '../../app/slices/commentSlice';
+
+type Props = {
+  comment: Comment;
+};
+export const CommentItem: FC<Props> = ({ comment }) => {
+  const dispatch = useAppDispatch();
+
+  const onDelete = () => {
+    dispatch(filterComment(comment));
+    dispatch(deleteCommentById(comment.id));
```

Comment:
Why do you need to dispatch twice?
I think you reducer can handle all the required updates together